### PR TITLE
`fastlane-env-reminder` is delivered as a container and run with docker

### PR DESCRIPTION
**Pull request by mollyIV**
### Description

Delivering `fastlane-env-reminder` as a container that can be run with docker will simplify the release process - no need for pushing the production dependencies 🎉 Please read more about how we release new versions [here](https://github.com/fastlane/github-actions/blob/update-documentation/CONTRIBUTING.md#how-to-release-new-versions).

The next step will be releasing a new version of this action and deleting production dependencies from `fastlane-env-reminder` on `latest` branch 👍 

closes https://github.com/fastlane/github-actions/issues/18

### Testing

To test the changes, please specify the following in your workflow file:

```yaml
  
name: Processing issues
on: 
  issues:
    types: [opened]

jobs:   
  fastlane-env:
    name: fastlane env reminder
    runs-on: ubuntu-latest    
    steps:
    - uses: actions/checkout@master
    - uses: mollyIV/github-actions/fastlane-env-reminder@fastlane-env-reminder-docker
      with:
        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

I tested the workflow on my private repository. 

image

image

🎊

### CODE REVIEW

File: fastlane-env-reminder/docs/contributors.md [hunk 1]
```diff
@@ -1,29 +0,0 @@
-# Contributors
```
Comment (janpio) [inline fastlane-env-reminder/docs/contributors.md]: Was this file be removed by accident?

File: fastlane-env-reminder/src/main.ts [hunk 2]
```diff
@@ -26,16 +28,16 @@ export async function run() {
...
     if (issueBody.includes('### Feature Request')) {
       console.log('The issue is a feature request, exiting');
       return;
     }
 
     const issueMessage = core.getInput('issue-message');
     const repoToken = core.getInput('repo-token', {required: true});
-    const issue: {owner: string, repo: string, number: number} = context.issue;
-    const client: github.GitHub = new github.GitHub(repoToken)
+    const issue: {owner: string; repo: string; number: number} = context.issue;
```
Comment (janpio) [inline fastlane-env-reminder/src/main.ts]: this doesn't look right initially - but my TS is... non existant.

**Comment by janpio** [comment]
The general idea sounds fine - if it works I am good with merging this.

File: fastlane-env-reminder/docs/contributors.md (hunk 1 above)
Comment (mollyIV) [reply fastlane-env-reminder/docs/contributors.md]: Nope, it was removed on purpose in favour of https://github.com/fastlane/github-actions/blob/master/CONTRIBUTING.md :wink:

File: fastlane-env-reminder/src/main.ts (hunk 2 above)
Comment (mollyIV) [reply fastlane-env-reminder/src/main.ts]: Yes, it takes a bit time to get used to TS syntax. 

If you look at the source code of `context.ts`: 

https://github.com/actions/toolkit/blob/5c894298f2f24441718eb049215aa6decd507193/packages/github/src/context.ts#L45-L52

you see that the issue json object is 1:1 with ours 👍

**Comment by mollyIV** [comment]
> The general idea sounds fine - if it works I am good with merging this.

The biggest benefit of running the action in docker container is simplified release process - no need for pushing the production dependencies 🎉 

Thanks for your review @janpio 🙇

**Pull request merged**